crates/forge/tests/fixtures/can_emit_block_number_in_trace.stderr

```diff
@@ -0,0 +1,19 @@
+Compiling 1 files
+ 
+Compiler run successful!
+
+Ran 4 tests for test/Contract.t.sol:BlockNumberTraceTest
+[FAIL. Reason: EvmError: Revert] testRevertFork() (block: 19626899) (gas: 9550)
+[FAIL. Reason: assertion failed] testRevertLocal() (gas: 3070)
+[PASS] testSuccessFork() (gas: 9598)
+[PASS] testSuccessLocal() (gas: 3048)
+Suite result: FAILED. 2 passed; 2 failed; 0 skipped; 
+
+Ran 1 test suite: 2 tests passed, 2 failed, 0 skipped (4 total tests)
+
+Failing tests:
+Encountered 2 failing testst/Contract.t.sol:BlockNumberTraceTest
```

Comment:
Strange formatting issue here, this should be "Encountered 2 failing tests in test/Contract.t.sol:BlockNumberTraceTest"